app/Filament/Resources/FlowMeasureResource/Pages/CreateFlowMeasure.php

```diff
@@ -2,20 +2,102 @@
 
 namespace App\Filament\Resources\FlowMeasureResource\Pages;
 
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+use App\Models\AirportGroup;
 use App\Enums\FlowMeasureType;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Collection;
+use Filament\Pages\Actions\Action;
+use App\Models\FlightInformationRegion;
 use Filament\Resources\Pages\CreateRecord;
 use App\Filament\Resources\FlowMeasureResource;
 use App\Helpers\FlowMeasureIdentifierGenerator;
-use App\Models\AirportGroup;
-use App\Models\FlightInformationRegion;
-use Illuminate\Support\Arr;
-use Illuminate\Support\Carbon;
-use Illuminate\Support\Collection;
 
 class CreateFlowMeasure extends CreateRecord
 {
     protected static string $resource = FlowMeasureResource::class;
 
+    private function hasMultipleAdeps(): bool
+    {
+        $adeps = Arr::get($this->data, 'adep');
+
+        if (count($adeps) > 1) {
+            return true;
+        }
+
+        foreach ($adeps as $adep) {
+            if (Arr::get($adep, 'value_type') === 'airport_group') {
+                return true;
+            }
+
+            if (Str::contains(Arr::get($adep, 'custom_value'), '*')) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    protected function getFormActions(): array
+    {
+        // TODO: Add tests for this
```

Comment:
Still need tests?

Reply:
Actually yes, however, I spent a little too much time on implementing this PR. Won't be able to add them today. I can add them later.